.github/workflows/cfc-validation.yml

```
@@ -231,6 +245,67 @@ jobs:
+          MODE=$(jq -r '.context.cfc.complianceMode // "advisory"' cdk.json)
+
+          echo "frameworks=$FRAMEWORKS" >> $GITHUB_OUTPUT
+          echo "mode=$MODE" >> $GITHUB_OUTPUT
+
+          echo "Compliance Frameworks: $FRAMEWORKS"
+          echo "Compliance Mode: $MODE"
+
+      - name: cfn-guard Validation - HIPAA
+        if: contains(steps.compliance.outputs.frameworks, 'HIPAA')
+        continue-on-error: ${{ steps.compliance.outputs.mode != 'enforce' }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:9:34: Double quote to prevent globbing and word splitting

(shellcheck)